docs/backend/upgrading/version-specific-migration/upgrade-to-61.md

```diff
@@ -117,3 +117,44 @@ If you have an existing Plone 5.2 or 6.0 site and you migrate to 6.1, then migra
 -   If the `plone.app.discussion` Python package is _not_ in your setup, but the site has existing comments (discussions), then the migration code stops with an error.
     Apparently you _were_ using comments in your site.
     Add the `plone.app.discussion` package to your dependencies, and run the migration again.
+
+
+## Distributions
+
+Plone 6.1 introduces the concept of a Plone {term}`distribution`.
+A Plone distribution is a Python package that defines specific features, themes, add-ons, and configurations that get activated when creating a Plone site.
+Now it is available in core Plone as the recommended way for {doc}`creating a new Plone site </install/add-site>`.
+
+```{seealso}
+For more information, see {doc}`/conceptual-guides/distributions`.
+```
+
+Distributions are optional.
+If your project only uses the `Products.CMFPlone` Python package, you can still create a Plone site in the old way.
+Consider, however, the following differences from distributions and from Plone 6.0.
```

Comment:
This sentence confused the heck out of me, and I only understood its intent after reading the list and already knowing some differences. I rewrote it for clarity. I hope it makes more sense now.